src/cabal2obs/Config/Ghc810x.hs

```diff
@@ -67,7 +67,9 @@ resolveConstraints = unwords ["cabal", "install", "--dry-run", "--minimize-confl
  -}
 
 constraintList :: ConstraintSet
-constraintList = [ "adjunctions"
+constraintList = [ "abstract-deque"
+                 ,"abstract-par"
```

Comment:
Please follow the formatting of the other entries.

Reply:
Are yiu reffering to a missing `space` in front of `"abstract-par"`?

Reply:
Yes.

Reply:
Thx, that must've got there by total mistake... :-) sorry for inconvenience!